packages/framework/src/Framework/Features/Navigation/NavigationItem.php

```diff
@@ -137,4 +137,14 @@ public static function normalizeGroupKey(?string $group): ?string
     {
         return $group ? Str::slug($group) : null;
     }
+
+    /** @experimental This feature may be removed before release, and the label will just fall back to the input $url */
+    protected static function makeTitleFromUrl(string $url): string
```

Comment:
This is probably going too far. I don't want to encourage people using direct links this much.

Reply:
Reverted in https://github.com/hydephp/develop/commit/8f2bd586a87f3e29a2c24978d53996b50d7655bd